src/matrixfree/solveIncrement.cc

```diff
@@ -388,11 +441,13 @@ void MatrixFreePDE<dim,degree>::solveIncrement(bool skip_time_dependent){
                 }
 
                 //check if solution is nan
-                if (!numbers::is_finite(solutionSet[fieldIndex]->l2_norm())){
-                    snprintf(buffer, sizeof(buffer), "ERROR: field '%s' solution is NAN. exiting.\n\n",
-                    fields[fieldIndex].name.c_str());
-                    pcout<<buffer;
-                    exit(-1);
+                if (currentIncrement%userInputs.skip_print_steps==0){
+                    if (!numbers::is_finite(solutionSet[fieldIndex]->l2_norm())){
+                        snprintf(buffer, sizeof(buffer), "ERROR: field '%s' solution is NAN. exiting.\n\n",
+                        fields[fieldIndex].name.c_str());
+                        pcout<<buffer;
+                        exit(-1);
+                    }
```

Comment:
I suggest reverting to checking every time step for the time being.

Reply:
Done